scaling_performance/network_optimization.adoc

```diff
@@ -67,6 +70,57 @@ xref:../architecture/core_concepts/routes.adoc#router-sharding[Router sharding]
 should be used to serve more routes towards applications and help horizontally
 scale the routing tier.
 
+[[scaling-performance-optimizing-router-haproxy]]
+=== Performance Optimizations
+
+[[scaling-performance-optimizing-router-haproxy-maxconn]]
+==== Setting the Maximum Number of Connections
+
+One of the most important tunable parameters for HAProxy is the 
+`maxconn` parameter which sets the maximum per-process number 
+of concurrent connections to a given number.  In {product-title} 
+HAProxy router this parameter is adjustable by editing 
+the 
+`xref:../install_config/router/default_haproxy_router.adoc#concurrent-connections[ROUTER_MAX_CONNECTIONS]`
+environment variable in the router's
+deployment configuration.
+
+
+[[scaling-performance-optimizing-router-haproxy-cpu-affinity]]
+==== CPU and Interrupt Affinity
+
+While it is entirely possible to run HAProxy with multiple worker
```

Comment:
Yes please, dropping the reference to multiple workers should remove any ambiguity.